apps/storefront/lib/auth/Readme.md

```diff
@@ -194,16 +194,16 @@ export const App = () => {
 
 ### How do I use it with Apollo?
 
-We have a hook for that called useApolloClient that authenticated fetch as its only argument and returns the current client and resetClient function
+We provide support for Apollo with a hook called `useApolloClient` that authenticates fetch as its only argument and returns the current client, as well as the `resetClient` function:
 
 ```javascript
 const { apolloClient, resetClient } = useApolloClient(saleorAuthClient.fetchWithAuth);
 ```
 
-Becasue we're using the client to also retrieve unauthenticated data in SSR, we separated them into two - one for SSR, that can be used outside of React flow, and the other returned by our hook.
+Because we're using the client to also retrieve unauthenticated data in SSR, we separated them into two - one for SSR, which can be used outside of React flow, and the other returned by our hook.
 
 ```javascript
-export const serverApolloClient = new ApolloClient({
+export const staticApolloClient = new ApolloClient({
```

Comment:
It's called `server` in the codebase, must have deviated while we're maintaining two codebases for this